src/main.ts

```diff
@@ -0,0 +1,43 @@
+import * as core from '@actions/core';
+import BumpPreview from 'bump-cli/lib/commands/preview';
+import BumpDeploy from 'bump-cli/lib/commands/deploy';
+
+async function run(): Promise<void> {
+  try {
+    const file: string = core.getInput('file');
+    const doc: string = core.getInput('doc');
+    const hub: string = core.getInput('hub');
+    const token: string = core.getInput('token');
+    const command: string = core.getInput('command') || 'deploy';
+    const cliParams = [file];
+    let deployCliParams = ['--doc', doc, '--token', token];
+
+    if (hub) {
+      deployCliParams = deployCliParams.concat(['--hub', hub]);
+    }
+
+    // debug is only output if you set the secret `ACTIONS_RUNNER_DEBUG` to true
+    core.debug(`Waiting for bump ${command} ...`);
+    core.debug(new Date().toTimeString());
+
+    switch (command) {
+      case 'preview':
+        await BumpPreview.run(cliParams);
+        break;
+      case 'validate':
```

Comment:
Do we cant to keep this `validate` command? I'm afraid that, as the `bump validate` call will have a different meaning, it's a bit confusing. Should we rename it into a `deploy-dry-run` (or anything else) one? WDYT?

Reply:
yeh your right. WDYT about simply `dry-run`?

Reply:
Works for me :+1:

Reply:
Do you mind if I leave the `validate` command to work? (removing it from the docs but let it launch a dry-run under the hood)

Reply:
:+1: